src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,85 @@
  */
 
 public class Plane {
+    private String nameOfCompany;
+    private String size;
+    private String type;
+    private int timeTravelled;
+    private int speed;
 
+    private Plane(PlaneBuilder builder) {
+        this.nameOfCompany = builder.nameOfCompany;
+        this.size = builder.size;
+        this.type = builder.type;
+        this.timeTravelled = builder.timeTravelled;
+        this.speed = builder.speed;
+    }
+
+    public String getNameOfCompany() {
+        return nameOfCompany;
+    }
+
+    public String getSize() {
+        return size;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public int getTimeTravelled() {
+        return timeTravelled;
+    }
+
+    public int getSpeed() {
+        return speed;
+    }
+
+    public class PlaneBuilder {
+        private String nameOfCompany;
+        private String size;
+        private String type;
+        private int timeTravelled;
+        private int speed;
+
+        public PlaneBuilder() {
+        }
+
+        public PlaneBuilder setNameOfCompany(String nameOfCompany) {
+            Plane.this.nameOfCompany = nameOfCompany;
```

Comment:
you should change builder's fields, not Plane's

Reply:
yes of course we need it, How without constructor we can create PlaneBuilder?